src/AugmintToken.test.js

```diff
@@ -1,5 +1,5 @@
+const loadEnv = require("./utils/env.js")();
```

Comment:
don't you get eslint warnings of unused variables? 
maybe you should set up eslint in your editor? also we could add eslint to pre-commit hook? or build..